src/component/LandingPageV2/FP1/style.css

```diff
@@ -284,14 +286,17 @@ h1 {
     margin-left: 13vh;
   }
 }
+*/
 
-@media only screen and (max-device-width: 650px) {
+@media (max-width: 650px) or (max-device-width: 650px) {
   /*.parent {*/
   /*  flex-direction: column;*/
   /*}*/
   .column-1{
+    /*
     margin-left: -3vh;
     margin-top: -3vh;
+    */
   }
 
   .client-blurb#fp1-client-blurb {
```

Comment:
For some reason, on XR this section is messed up - it looks like its not contained correctly by the surrounding parent (same in fp2)